resources/migrations/001_update_migrations.yaml

```diff
@@ -9288,6 +9288,252 @@ databaseChangeLog:
                   name: type
             indexName: idx_collection_type
 
+  - changeSet:
+      id: v51.2024-09-26T03:01:00
+      author: escherize
+      comment: iff enable-embedding is set, copy -> enable-embedding-interactive
+      preConditions:
+        - onFail: MARK_RAN
+        - or:
+            - and:
+                - dbms:
+                    type: postgresql
+                - sqlCheck:
+                    expectedResult: 1
+                    sql: SELECT count(*) FROM setting WHERE key = 'enable-embedding';
```

Comment:
this setting is encrypted, you'll need to decrypt it first. see `v50.2024-06-12T12:33:07` for inspiration

Reply:
well, he just does `count(*)` and settings name is not encrypted, so should be fine?

Reply:
I think I might be missing something here: Can you spell out what the issue is going to be? I am selecting on the non-encrypted `key` column, and copying the (possibly-encrypted) `value` column into another (possibly-encrypted) `value` column. 

Copying an encrypted value into an identical value shouldn't be a problem, right?

Reply:
Trying to see what mean, and looking into it some more: the decryption only depends on the value of [`MB_ENCRYPTION_SECRET_KEY`, and `v`](https://github.com/metabase/metabase/blob/sdk-embedding-settings-part-two/src/metabase/util/encryption.clj#L150-L151), so I don't see why it wouldn't just decrypt like the original setting.

Reply:
oh right, for some reasons I remember you checked the value of this setting somewhere, maybe I'm blind :D. all are good, then